src/app/profile/yourGiving/givingRecipientView/donationRecipient/donationRecipient.tpl.html

```diff
@@ -0,0 +1,88 @@
+<div class="panel panel-margin-bottom recent-recipient-row">
+  <div class="panel-body">
+
+    <!-- Recipient Row -->
+    <div class="row">
+      <div class="col-md-2 col-full-height col-sm-12 col-xs-12 mb">
+        <img class="full-width" desig-src="{{$ctrl.recipient['designation-number']}}">
+      </div>
+      <div class="col-sm-12 col-md-7 col-full-height">
+        <h4 class="mb0">{{$ctrl.recipient['designation-name']}}</h4>
```

Comment:
The designation number seems to be repeated in the name field. Seems to be an api issue. Could be my old data...